src/nvim/lua/vim.lua

```diff
@@ -187,10 +189,9 @@ end
 ---                - 2: continues the paste (zero or more times)
 ---                - 3: ends the paste (exactly once)
 --@returns false if client should cancel the paste.
-local function paste(lines, phase) end  -- luacheck: no unused
```

Comment:
this was here for the doc generator...

Reply:
It seems to work fine without it, unless I'm missing something. If I run scripts/gen_vimdoc, that part is unchanged.

Reply:
oh wait the doc part wasn't even attached to function. I'm probably generating docs incorrectly.